clustershell.spec.in

```diff
@@ -192,7 +203,12 @@ rm -rf %{buildroot}
 %if 0%{?rhel}
 %defattr(-,root,root,-)
 %endif
-%doc ChangeLog COPYING.LGPLv2.1 README.md
+%doc ChangeLog README.md
+%if %{suse_version} >= 1500
```

Comment:
Typo, should be:
```%if 0%{?suse_version} >= 1500```

Reply:
Oops, yes. Will update patch, sorry.

Reply:
I've pushed a new version of the spec file.